modules/nodes-nodes-jobs-about.adoc

```diff
@@ -0,0 +1,109 @@
+link:https://www.openshift.com/pricing/index.html[pricing page].
+====
+endif::[]
+
+[WARNING]
+====
+A CronJob creates a job object approximately once per execution time of its
+schedule, but there are circumstances in which it fails to create a job or
+two jobs might be created.  Therefore, jobs must be idempotent and you must
+configure history limits.
+====
+
+= About creating jobs
+
+Both types of jobs require a job configuration that consists of the following key parts:
+
+- A pod template, which describes the application the pod will create.
+- An optional `parallelism` parameter, which specifies how many pod replicas running in parallel should execute a job. If not specified, this defaults to
+ the value in the `completions` parameter.
+- An optional `completions` parameter, specifying how many successful pod completions are needed to finish a job. If not specified, this value defaults to one.
+
+== About setting a maximum duration for jobs
+
+When defining a job, you can define its maximum duration by setting
+the `activeDeadlineSeconds` field. It is specified in seconds and is not
+set by default. When not set, there is no maximum duration enforced.
+
+The maximum duration is counted from the time when a first pod gets scheduled in
+the system, and defines how long a job can be active. It tracks overall time of
+an execution and is irrelevant to the number of completions (number of pod replicas
+needed to execute a task). After reaching the specified timeout, the job is
+terminated by {product-title}.
+
+== About setting a job back off policy for pod failure
+
+A Job can be considered failed, after a set amount of retries due to a
+logical error in configuration or other similar reasons. Failed Pods associated with the Job are recreated by the controller with
+an exponential back off delay (`10s`, `20s`, `40s` …) capped at six minutes. The
+limit is reset if no new failed pods appear between controller checks.
+
+Use the `spec.backoffLimit` parameter to set the number of retries for a job.
+
+== About configuring a CronJob to remove artifacts
+
+CronJobs can leave behind artifact resources such as jobs or pods.  As a user it is important
+to configure history limits so that old jobs and their pods are properly cleaned.  There are two fields within CronJob's spec responsible for that:
+
+* `.spec.successfulJobsHistoryLimit`. The number of successful finished jobs to retain (defaults to 3).
+
+* `.spec.failedJobsHistoryLimit`. The number of failed finished jobs to retain (defaults to 1). 
+
+[TIP]
+=====
+Delete CronJobs that you no longer need:
+----
+$ oc delete cronjob/<cron_job_name>
+----
+Doing this prevents them from generating unnecessary artifacts.
+=====
+
+= About scaling jobs
+
+A job can be scaled up or down, which, in the case of jobs, modifies the
+`spec.parallelism` parameter. This will result in modifying the number of pod
+replicas running in parallel, executing a job.
```

Comment:
Drop this point, we'll be removing this functionality. It's confusing b/c it doesn't work the same way as with deployments and other workload controllers.